boards/arm/stm32/nucleo-g431kb/src/stm32_dac.c

```diff
@@ -80,6 +84,25 @@ int stm32_dac_setup(void)
       return ret;
     }
 
+#endif
+
+#ifdef CONFIG_STM32_DAC3CH2
+  g_dac5 = stm32_dacinitialize(5);
+  if (g_dac5 == NULL)
+    {
+      aerr("ERROR: Failed to get DAC interface\n");
+      return -ENODEV;
+    }
+
+  /* Register the DAC driver at "/dev/dac0" */
+
+  ret = dac_register("/dev/dac0", g_dac5);
```

Comment:
What will happen if CONFIG_STM32_DAC1CH1 and CONFIG_STM32_DAC3CH2 be defined? I think it will fail to register the second "/dev/dac0". I think you need to avoid it.

Reply:
You are right. I will change the register path of DAC3CH2 to "/dev/dac5".

Thanks